force-app/main/default/classes/newTaskTraining.cls

```diff
@@ -0,0 +1,18 @@
+public class newTaskTraining {
+    public static void createTask() {
+        Date today = Date.today();
+        List<Training__c> endedTrains = [SELECT Name, End_Date__c, Training_Course__r.OwnerId
+                                        FROM Training__c
+                                        WHERE DAY_ONLY(End_Date__c) = :today];
+        //System.debug(endedTrains);
+        List<Task> taski = new List<Task>();
+        for(Training__c train :endedTrains) {
+            Task task = new Task(Subject = 'Fill GPA (marks) and mark all participants as Participated.',
+                                Status = 'Not Started',
```

Comment:
Please move all the hardcoded values to the top of the page and make them constants. So that you can reuse it in other methods or test classes and not hardcode every time. 

https://www.tutorialspoint.com/apex/apex_constants.htm#:~:text=As%20in%20any%20other%20programming,value%20throughout%20the%20program%20execution.

https://developer.salesforce.com/docs/atlas.en-us.232.0.apexcode.meta/apexcode/langCon_apex_constants.htm